pkg/secrets/secrets_test.go

```diff
@@ -166,6 +166,20 @@ func TestAddReplaceSecretName(t *testing.T) {
 
 	_, err = manager.Store("mysecret", []byte("mydata.diff"), drivertype, storeOpts)
 	require.NoError(t, err)
+
+	_, data, err := manager.LookupSecretData("mysecret")
+	require.NoError(t, err)
+	require.Equal(t, string(data), "mydata.diff")
+
+	_, err = manager.Store("nonexistingsecret", []byte("mydata"), drivertype, storeOpts)
+	require.NoError(t, err)
+
+	storeOpts.Replace = false
+	_, err = manager.Store("nonexistingsecret", []byte("newdata"), drivertype, storeOpts)
+	require.Error(t, err)
+
+	_, err = manager.Delete("nonexistingsecret")
+	require.NoError(t, err)
```

Comment:
Unless I am doing something wrong this test passes with and without your changes so it does not check what you think it does I guess

Reply:
These are good things to check, but I agree, I don't see a fix in my code.